app/src/main/java/org/simple/clinic/editpatient/EditPatientInit.kt

```diff
@@ -21,7 +24,7 @@ class EditPatientInit(
         LoadInputFields,
     )
 
-    if (!model.hasColonyOrVillagesList) {
+    if (!model.hasColonyOrVillagesList && features.isEnabled(Feature.VillageTypeAhead)) {
```

Comment:
We are introducing an impurity here. Can we instead pass the flag in as a `Boolean` constructor property instead?